_data/navigation.yml

```diff
@@ -273,6 +273,8 @@ wiki:
         url: /wiki/tools/stream-rviz/
       - title: Web-Based Visualization using ROS JavaScript Library
         url: /wiki/tools/roslibjs/
+      - title: MapViz for Map Based Visualization in ROS2
+         url: /wiki/planning/mapviz/
```

Comment:
Whitespace. `url:` should be aligned with title. Some YAML parsers will throw an error if not. 

You're using spaces, right?

CC: @nevalsar